src/50_Samples_%26_Templates/Product.html

```diff
@@ -387,7 +387,9 @@ <h2 class="m1">Apple</h2>
    <!-- ## Image Gallery -->
   <!-- The `amp-carousel` component works very well for product image galleries. Learn how the `amp-carousel` component works [here](/advanced/image_galleries_with_amp-carousel/).
 
-  Notice how we are binding to the `slide` and `class` by using to the values of the variables `selectedSlide` and `selectedColor`. -->
+  Notice how we are binding to the `slide` and `class` by using to the values of the variables `selectedSlide` and `selectedColor`. 
+  It's not possible to set a default value for variables, for example we use the expression`(!selectedColor || selectedColor == 'green')`
```

Comment:
I trieI tried to update the code this way:
`on="tap:AMP.setState({product.selectedSlide: 0})"`
and I get:

> cdn.ampproject.org/v0/amp-bind-0.1.js:22 amp-bind: AMP.setState() failed with error: Parse error on line 1:
> {product.selectedSlide: 0}
> --------^
> Expecting ':', got '.'​​​

Is it supported to set variables from the json state in the `setState`? I remember this was not happening at the start and this is why I was using global variables.